scripts/copy-declarations.js

```diff
@@ -0,0 +1,31 @@
+// Copy manually-written `.d.ts` files from `src/` to `types/src/`
+import { copyFileSync } from "node:fs";
```

Comment:
Sorry, missed the import in my previous suggestion.
```suggestion
import { copyFileSync, mkdirSync } from "node:fs";
```